src/widgetastic_patternfly/__init__.py

```diff
@@ -1579,6 +1579,11 @@ def has_item(self, item):
         """
         return item in self.items
 
+    @property
+    def selected_option(self):
```

Comment:
Q: what is returned by read() call for this widget ?

Reply:
No; with read its not working... please give me some time...